.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            2
+        ],
+        "quotes": [
+            2,
+            "single"
```

Comment:
Same with some other files we have that I guess we don't bother linting, like package.json.